lib/msf/core/module/alert.rb

```diff
@@ -202,9 +202,10 @@ def add_alert(level, msg, &block)
   def alert_user
     self.you_have_been_warned ||= {}
 
-    errors.each do |msg|
+    errors.each do |msg| 
+      # require 'pry-byebug'; binding.pry
       if msg && !self.you_have_been_warned[msg.hash]
-        print_error(msg)
+        print_error(msg.full_message)
```

Comment:
I think the existing API assumes that a user will register strings with this API:

https://github.com/rapid7/metasploit-framework/blob/ad4b47f6ac6b3dfb9b55841c24d6bfb930e800d5/lib/msf/core/module/alert.rb#L13-L41

I'm not sure how an actual Ruby error object got registered here, but I think you might be fixing a side effect of a different bug here rather than fixing the actual bug 👀